.github/workflows/push.yaml

```diff
@@ -73,7 +73,8 @@ jobs:
           OCP-4.13,
           OCP-4.14,
           OCP-4.15,
-          OCP-4.16
+          OCP-4.16,
+          OCP-4.17
```

Comment:
Went ahead and changed to use the wildcard that was added to the dci-component GHA, this should take care of creating the components in the all the last OCP versions

Reply:
It's fine for me, but I cannot accept a change from myself, waiting for your review to finally merge this.